ocs_ci/ocs/resources/rgw.py

```diff
@@ -48,22 +48,27 @@ def get_credentials(self, secret_name=constants.NOOBAA_OBJECTSTOREUSER_SECRET):
 
         """
         secret_ocp_obj = OCP(kind=constants.SECRET, namespace=self.namespace)
-        route_ocp_obj = OCP(
-            kind=constants.ROUTE, namespace=config.ENV_DATA["cluster_namespace"]
-        )
 
         if storagecluster_independent_check():
-            endpoint = route_ocp_obj.get(
-                resource_name=constants.RGW_SERVICE_EXTERNAL_MODE
-            )
             secret_name = constants.EXTERNAL_MODE_NOOBAA_OBJECTSTOREUSER_SECRET
+            cos_ocp_obj = OCP(
```

Comment:
cos could be misinterpreted as IBM COS (Cloud Object Storage). Can we change the var name?